ingestors/kafka/src/main/scala/hydra/kafka/endpoints/TopicMetadataEndpoint.scala

```diff
@@ -169,11 +169,11 @@ class TopicMetadataEndpoint[F[_]: Futurable](consumerProxy:ActorSelection,
                       val req = TopicMetadataV2Request.fromMetadataOnlyRequest(schemas, mor)
                       onComplete(
                         Futurable[F].unsafeToFuture(createTopicProgram
-                          .publishMetadata(t, req))
+                          .createTopicFromMetadataOnly(t, req))
                       ) {
                         case Failure(exception) =>
                           addHttpMetric(topic, StatusCodes.InternalServerError, "/v2/metadata", startTime, method.value)
```

Comment:
If it is a bad request, need to change this line

Reply:
I believe it would always be a Bad Request. createTopic on line 239 only returns BadRequest errors, so putV2Metadata should match the behavior